web/themes/custom/cloudy/templates/content/council-document/node--council-document--embedded.html.twig

```diff
@@ -107,4 +107,5 @@
   {{ content.field_neighborhood }}
   {{ content.field_introduced_by }}
   {{ content.field_bureau }}
+  {{ content.field_service_area }}
```

Comment:
This can probably be removed since the service area is hidden from this view mode.